src/ecmult_const_impl.h

```diff
@@ -15,8 +15,9 @@
 /* This is like `ECMULT_TABLE_GET_GE` but is constant time */
 #define ECMULT_CONST_TABLE_GET_GE(r,pre,n,w) do { \
     int m; \
-    int abs_n = (n) * (((n) > 0) * 2 - 1); \
-    int idx_n = abs_n / 2; \
+    int mask = (n) >> (sizeof(n) * CHAR_BIT - 1); \
```

Comment:
Maybe add a comment here explaining what it does (or give the old code as explanation)?

Reply:
OK.